src/notation/imasternotation.h

```diff
@@ -54,6 +54,8 @@ class IMasterNotation
     virtual void removeExcerpts(const ExcerptNotationList& excerpts) = 0;
     virtual void sortExcerpts(ExcerptNotationList& excerpts) = 0;
 
+    virtual IExcerptNotationPtr resetReplaceExcerpt(IExcerptNotationPtr excerpt) = 0;
```

Comment:
A clearer interface would look like this:

`virtual void resetExcerpt(IExcerptNotationPtr excerpt) = 0:`

Or

`virtual void resetExcerptToDefault(IExcerptNotationPtr excerpt) = 0:`

Reply:
But that would mean we have to reset the `IExcerptNotationPtr` while leaving it in place. That is potentially a lot more work and prone to make things go out of sync. Just replacing it with a fresh one was a lot more straight forward. But if you think it's better I'll go for it.

Reply:
> That is potentially a lot more work

This is not a very good argument. Looking at this interface, I can't tell what it does. I have to open the .cpp file to understand that.

> Just replacing it with a fresh one was a lot more straight forward.

You can still destroy the old engraving::Excerpt*  and replace it with a fresh one inside the implementation of the function. What kind of difficulties does this cause?

Reply:
The big problem of that approach is that it requires us to change the `Score*` of an existing `Notation`. At least I have never actively kept in mind that we would ever have to do that, so who knows where we accidentally store references to the old `Score*`.

At a first glance, it seems to work fine though, so we can try. But to me it still feels like a risky thing to do.

By the way, my implementation was somewhat inspired by `MasterNotation::updateExcerpts()`.